src/pages/CallBack.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+
+import { TP } from '../components/THeadings';
+import TProgressCircle from '../components/TProgressCircle';
+import { SdkErrors, throwError, UserApps } from 'tonomy-id-sdk';
+import './main.css';
+
+export default function CallBackPage() {
+    useEffect(() => {
+        const params = new URLSearchParams(window.location.search);
```

Comment:
in `UserApps.onRedirectLogin()` We take care of the URLSearchParams. We should do the same here. Both for consistency and also so that developers need to do less